config-service-api/src/main/proto/org/hypertrace/config/service/v1/config_service.proto

```diff
@@ -39,6 +39,12 @@ message UpsertConfigRequest {
 message UpsertConfigResponse {
   // config value upserted into the store
   google.protobuf.Value config = 1;
+
+  // timestamp (in milliseconds since epoch) at which this config was created
+  int64 creation_timestamp = 2;
```

Comment:
When it comes to code it feel ambiguous to not to know the unit. How about creation_timestamp_ms?

Reply:
I completely agree, however here I'd prefer consistency, since the same field is already present in other responses in the same service API. For what it's worth, I believe across the entire hypertrace platform timestamps stored as long are always millis (and should have been named that way in the first place) - we don't use seconds or micro/nano anywhere that I'm aware of.